README.md

```diff
@@ -41,49 +43,38 @@ cd Generals-RL
 pip install -e .
 ```
 
-## Usage example (🤸 Gymnasium)
-
+## 🚀 Getting Started
+We create agents in two modes:
+- **classic mode** - this mode is intended for agents based on classical algorithms and heuristics.
+  In this mode you can start by subclassing an `Agent` class just like [`RandomAgent`](./generals/agents/random_agent.py) or [`ExpanderAgent`](./generals/agents/expander_agent.py).
+  You can specify your agent `id` (name) and `color` and the only thing remaining is to implement the `act` function, that has the signature explained in sections down below.
+- **learning mode** - this mode is more "open" and we expect that you have some experience with **Gymnasium** or **PettingZoo**.
```

Comment:
"We create..." -> "You can create ...". Maybe reduce usage of "can", since they indeed must subclass `Agent` class, right? Is `id`/`color` mandatory or not?

Reply:
"and we expect" -> "it is expected"

Reply:
I have redone this part completely, hope its ok like this. More will come in wiki. wdyt?

Reply:
sound good